### ISSUE TITLE
[3.0] Meeting can't be joined when another 3.0-Server was used before in the same Browser session

### ISSUE THREAD

**Issue by Ithanil** [bigbluebutton/bigbluebutton#21927]
**Describe the bug**
Ever since we are actively using BBB 3.0 and run multiple servers, me and my colleagues regularly had problems joining sessions with errors like seen in the screenshot below. When the problem occurs, the only way to resolve it is to restart the browser session. The only way to prevent it is to run only a single BBB-3.0 server.

Now, during work on beta.7, I got reminded of this problem again.

**To Reproduce**
Steps to reproduce the behavior (tested with Chromium):
1. Enter a meeting on one server.
2. Leave the meeting. 
3. Enter a meeting on a different server.

**Screenshots**
![Screenshot From 2024-12-18 09-40-23](https://github.com/user-attachments/assets/0ce0708b-ed4e-415d-9b46-62a6ff0430a6)

**BBB version:**
Any BBB 3.0 version.

**Desktop (please complete the following information):**
- Linux Chromium
- 
**Additional context**
BBB 3.0 stores the key `BBB_bbbWebApi_index` in the Session storage, with content like:
```
{returncode: "SUCCESS", version: "2.0", apiVersion: "2.0", bbbVersion: "3.0.0-beta.7",…}
apiVersion
: 
"2.0"
bbbVersion
: 
"3.0.0-beta.7"
graphqlApiUrl
: 
"https://bbb-xyz-123/api/rest"
graphqlWebsocketUrl
: 
"wss://bbb-xyz-123/graphql"
returncode
: 
"SUCCESS"
version
: 
"2.0"
```
I suspect this is related to the problem. BBB 2.7 didn't store anything  like that, especially nothing containing the server FQDN.

**Comment by ffdixon** [comment]
Thanks for reporting this (and it's never too late :-) )

**Comment by antobinary** [comment]
Is this (group of) servers using cluster proxy?

**Comment by Ithanil** [comment]
> Is this (group of) servers using cluster proxy?

Yes it is. Sorry, forgot to mention that. But also it didn't look like a typical "Cluster Proxy issue" to me.

**Comment by gustavotrott** [comment]
Hi @Ithanil,

I’ve updated the way the data is being persisted. I’d appreciate it if you could provide feedback after setting up a server with the BBB version that includes my fix.

Thanks!

**Comment by Ithanil** [comment]
@gustavotrott Thanks for the quick reaction! Unfortunately, with your commit in use, join now results in a 404 response to a request to  `https://proxy/bbb-xyz/html5client/?sessionToken=token` (request goes through proxy and BBB server responds 404).

**Comment by Ithanil** [comment]
@gustavotrott Wait, gimme a moment to verify that it's not just because I've upgraded bbb-html5 without re-applying some customization afterwards.

**Comment by gustavotrott** [comment]
> @gustavotrott Sorry for the noise, of course the 404 was my fault. I didn't consider that upgrading the bbb-html5 package would replace `/usr/share/bigbluebutton/nginx/bbb-html5.nginx.static` .
> 
> Your fix appears to work perfectly fine, thank you!

Awesome! Thank you for confirming it.
